src/components/App.tsx

```diff
@@ -27,9 +47,72 @@ const App: React.FC = () => {
           <Grid.Column stretched width={12}>
             <Segment>
               <Switch>
-                <Route path="/" exact component={Home} />
-                <Route path="/form" component={Register} />
-                <Route path="/applications" component={ApplicationsList} />
+                <Route exact path="/">
+                  <Home />
+                </Route>
+                <Route exact path="/login">
+                  <Login />
+                </Route>
+                <Route path="/applications">
+                  <ApplicationsList />
+                </Route>
+                <Route path="/application/new">
+                  <ApplicationNew />
+                </Route>
+                <Route exact path="/application/:appId">
+                  <Application />
+                </Route>
+                <Route exact path="/application/:appId/:sectionName/p:page">
+                  <Application />
+                </Route>
+                <Route exact path="/application/:appId/summary">
+                  <Application summary={true} />
+                </Route>
+                <Route exact path="/application/:appId/approval">
+                  <Approval />
+                </Route>
+                <Route exact path="/admin">
+                  <Admin />
+                </Route>
+                <Route exact path="/admin/templates">
+                  <TemplateList />
+                </Route>
+                <Route exact path="/admin/template/new">
+                  <TemplateNew />
+                </Route>
+                <Route exact path="/admin/template/:templateId/:step">
+                  <Template />
+                </Route>
+                <Route exact path="/admin/users">
+                  <AdminUsers />
+                </Route>
+                <Route exact path="/admin/permissions">
+                  <AdminPermissions />
+                </Route>
+                <Route exact path="/admin/config">
+                  <Config />
+                </Route>
+                <Route exact path="/account">
+                  <Account />
+                </Route>
+                <Route exact path="/organisation/:orgName">
+                  <Organisation />
+                </Route>
+                <Route exact path="/organisation/:orgName/member-edit">
```

Comment:
I think this path is odd. I would expect something as `/organisation/:orgName/members` and the action to edit member would be just a option in there. Maybe something that the user could edit directly in there and then save it. Not sure if 1. we need a separated page for this and 2. If we would allow adding/removing then this path would be with a different meaning, or we would have one per action. For now, I think it could just be called members.

Reply:
Yup, good call. Have changed it to just `/members` for now.